gui/slick/interfaces/default/home_print.mako

```diff
@@ -0,0 +1,203 @@
+<%!
+    from sickbeard import db, network_timezones, sbdatetime
+    from sickbeard.common import ARCHIVED, FAILED, SKIPPED, UNAIRED, WANTED
+    from sickbeard.common import Quality, qualityPresets, qualityPresetStrings
+
+    import datetime
+    import re
+    import sickbeard
+%>
+
+<%
+    myDB = db.DBConnection()
+    today = str(datetime.date.today().toordinal())
+
+    status_quality  = '(' + ','.join([str(x) for x in Quality.SNATCHED + Quality.SNATCHED_PROPER]) + ')'
+    status_download = '(' + ','.join([str(x) for x in Quality.DOWNLOADED + [ARCHIVED]]) + ')'
+
+    sql_statement  = 'SELECT showid, '
```

Comment:
This query comes from `home.mako`.
I didn't have the courage to check if it does more than what is needed here.